freeciv-web/src/main/webapp/javascript/hotseat.js

```diff
@@ -235,7 +235,9 @@ function show_hotseat_new_phase()
 
   if (overview_active) $("#game_overview_panel").parent().hide();
   $("#game_unit_panel").parent().hide();
-  setTimeout("$('#game_unit_panel').parent().hide();", 1000);
+  setTimeout(function() {
```

Comment:
Thanks for this pull-request!!

I don't think this particular change is correct, it does not do the same thing. Please put $('#game_unit_panel').parent().hide();  into a new function, and call setTimeout on that function.